src/unk_0200ACF0.c

```diff
@@ -0,0 +1,223 @@
+#include "global.h"
+#include "unk_0200ACF0.h"
+#include "unk_020215A0.h"
+#include "unk_02022588.h"
+
+BOOL sub_0200ACF0(GF_2DGfxResObj *obj) {
+    GF_ASSERT(obj != NULL);
+    GF_ASSERT(GF2DGfxResObj_GetResType(obj) == GF_GFX_RES_TYPE_CHAR);
+
+    UnkStruct_02021758 sp00;
+    sp00.charData = GF2DGfxResObj_GetCharDataPtr(obj);
```

Comment:
sp notation

Reply:
etc for file

Reply:
keeping it in place for objects of type (pointer to) UnkStruct_XXXXXXXX